pkg/api/types.go

```diff
@@ -1920,6 +1920,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// If specified, the pod's tolerations
+	// +optional
+	Tolerations []Toleration
```

Comment:
Maybe not accurate to use `pair` here, as both those two fields are `optional`, I want to say that we should have both `Taint for NodeSpec` and `Tolerations for Pod` set if want to use this feature. ;-)